### PR TITLE
Add InstallmentInfos

### PR DESCRIPTION
**Pull request by fire-at-will**
This PR adds `InstallmentInfos` to the KMP SDK.

### CODE REVIEW

File: core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/models/SubscriptionOption.ios.kt [hunk 1]
```diff
@@ -9,4 +9,5 @@ public actual interface SubscriptionOption {
     public actual val presentedOfferingIdentifier: String?
     public actual val presentedOfferingContext: PresentedOfferingContext?
     public actual val purchasingData: PurchasingData
+    public actual val installmentsInfo: InstallmentsInfo?
```
Comment (fire-at-will) [inline core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/models/SubscriptionOption.ios.kt:12]: I've added this as an optional for iOS when I know it'll always be null. Do I need to add anything to make sure it's being correctly parsed as `null` when it's being parsed out from PHC?

Comment (tonidero) [reply core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/models/SubscriptionOption.ios.kt:12]: The `SubscriptionOption` object itself should always be null in iOS, so I don't think we need to do any extra parsing.

**Review by tonidero — APPROVED** [review]
I believe this should work! Maybe good to let @JayShortway also take a look just in case

File: core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/models/SubscriptionOption.ios.kt (hunk 1 above)
Comment (fire-at-will) [reply core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/models/SubscriptionOption.ios.kt:12]: I just tested making a purchase through a paywall in iOS through the KMP purchase tester app and everything worked correctly, so I think we're probably good here.

**Review by JayShortway — APPROVED** [review]
LGTM!

**Pull request merged**